src/app/app.component.html

```diff
@@ -580,4 +587,34 @@ <h3 class="modal-title">Change Password</h3>
       </button>
     </div>
   </clr-modal>
+  <clr-modal [(clrModalOpen)]="alertDeleteAccessCodeModal">
+    <h3 class="modal-title">Are you sure you want to delete?</h3>
```

Comment:
Make clear what is being deleted :
Display the selected access codes again, or the number of selected access codes 
„Are you sure you want to delete 7 access codes?“